pkg/skaffold/deploy/helm_test.go

```diff
@@ -374,6 +397,30 @@ func TestHelmDeploy(t *testing.T) {
 			runContext: makeRunContext(testDeployHelmStyleConfig, false),
 			builds:     testBuilds,
 		},
+		{
+			description: "helm image strategy with explicit registry should set the Helm registry value",
+			cmd: &MockHelm{
+				t:         t,
+				getResult: fmt.Errorf("not found"),
+				installMatcher: func(cmd *exec.Cmd) bool {
+					dockerRef, err := docker.ParseReference(testBuilds[0].Tag)
+					if err != nil {
+						return false
+					}
+
+					expected := fmt.Sprintf("image.registry=%s,image.repository=%s,image.tag=%s", dockerRef.Domain, dockerRef.Path, dockerRef.Tag)
```

Comment:
please modify this to the actual strings instead of relying on parsing the the reference. It will be easier to understand the behavior with explicit examples instead of transformed logic.

Reply:
Changed!